OpenSim/Application.swift

```diff
@@ -24,12 +24,12 @@ final class Application {
     static let sizeDispatchQueue = DispatchQueue(label: "com.pop-tap.size", attributes: .concurrent, target: nil)
     
     var sandboxUrl: URL? {
-        guard let url = device.containerURLForApplication(self),
-            FileManager.default.fileExists(atPath: url.path)
+        guard let safeurl = device.containerURLForApplication(self),
+            FileManager.default.fileExists(atPath: safeurl.path)
             else {
-                return nil
+                return url
```

Comment:
Sorry but I don't quite follow here. Why is it returning the app's directory for sandbox's directory?